src/com/lundekhan/api/ApiRoute.kt

```diff
@@ -43,11 +44,21 @@ fun Route.apiRoute(redirections: MutableMap<String, String>): Route = route("/ap
         call.respond(splitBills(payments))
     }
 
+    /**
+     * POST: /api/stokk
+     */
+    post("/stokk") {
+        val compound = call.receiveOrNull<CompoundInput>()
```

Comment:
Prefer one-liner personally.

Reply:
Not quite sure what it does, I can remove it and everything stil works as suspected